runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -323,17 +329,20 @@ private void updateWarning(boolean weaponSwitch)
 				{
 					if (loginWarningMessageCount == 3)
 					{
-						sendChatMessage("<col=871A1A>[ATTACK STYLE]</col> You logged in wielding a weapon that will grant you " + returnAttackColor(skill) + " XP. #" + loginWarningMessageCount);
+						if (config.warningMessages())
```

Comment:
There should not be three `if (config.warningMessages())` checks for a block which will always print a chat message of some kind. Ideally, this check would be done in `sendChatMessage` (which should probably be renamed accordingly) so it does not always need a paired config check to determine whether it should print.